Makefile

```diff
@@ -15,6 +17,14 @@ release:
 	git tag -a -m "Release $(VERSION)" v$(VERSION); \
 	printf "\nNew release $(VERSION) tagged!\n\n"
 
+run-collector:
+	echo $(WARNING)
```

Comment:
I would prepend `@` to this line (and for the same line in run-telemetryui) so that the warning isn't printed twice.